### DISCUSSION TITLE
Reject files that don't fit a scheme

### DISCUSSION THREAD

**Discussion by FreshImmuc** [QuorumDMS/ftp-srv#329] [Q&A]
how do i reject file uploads at all?

**Comment by matt-forster** [comment]
That would be up to you - but you can throw an error and pass back the rejected signal; here is an example from the default STOR command:

https://github.com/QuorumDMS/ftp-srv/blob/50e8c455d621c7cb069b6e22fff1f6462a9d03b6/src/commands/registration/stor.js#L61-L65
